File: app/src/main/java/app/we/go/emojidraw/model/Stroke.java
package app.we.go.emojidraw.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Represents a stroke that the user has drawn.
 */
public class Stroke {
    private final List<Integer> xcoords;
    private final List<Integer> ycoords;

    public Stroke() {
        xcoords = new ArrayList<>();
        ycoords = new ArrayList<>();
    }

    public void addPoint(int x, int y) {
        xcoords.add(x);
        ycoords.add(y);
    }

    public List<Integer> getXcoords() {
        return xcoords;
    }

    public List<Integer> getYcoords() {
        return ycoords;
    }
}

File: app/src/main/java/app/we/go/emojidraw/model/Stroke.kt
package app.we.go.emojidraw.model

class Stroke {

    val xcoords = mutableListOf<Int>()
    val ycoords = mutableListOf<Int>()

    fun addPoint(x: Int, y: Int) {
        xcoords.add(x)
        ycoords.add(y)
    }
}